src/assets/mixins.scss

```diff
@@ -13,9 +13,11 @@
     padding-top: ($width / $height) * 100%;
   }
 
-  > img,
-  div {
-    @apply absolute w-full h-full inset-0 object-cover;
+  @if $affectChildren {
+    > img,
+    div {
+      @apply absolute w-full h-full inset-0 object-cover;
+    }
```

Comment:
In my use-case, I had a direct descendant that was absolutely positioned, which this was affecting. I don't think this definition should exist here, but I didn't want to adjust existing uses of this mixin.